src/symmetrictensor.jl

```diff
@@ -1,10 +1,14 @@
-"""Type constructor
+"""
+    SymmetricTensor(frame, bls, bln, dats, sqr)
+
+Construct a symmetric tensor.
 
-frame - stores ArrayNArrays{T,N}
-bls - Int, size of ordinary block
-bln - Int, number of blocks
-datasize - Int, size of data stored (in each direction the same)
-sqr - Bool, is the last block size a same as ordinary's block size
+# Arguments
+- `frame::ArrayNArrays{T,N}`: the raw data.
+- `bls::Int`: the size of ordinary block.
+- `bln::Int`: the number of blocks.
+- `dats::Int`: the size of data stored (the same in each direction).
+- `sqr::Bool`: if the last block is squared.
 """
```

Comment:
My suggestion:

if the last block is squared -> if last blocks are squares (N-squares)


Comment:

in matrix form st looks like that
|x1  x2    y1 |
|      x3    y2 |
|               y' |
where y1, y2 (these are last blocks) may not be squares. For N-dimensional tensor the analog would not be 
N-arrays.

Reply:
Does "if the last few blocks are squares (N-squares)" work?